Makefile

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
lol, I just removed it (again) with my PR from yesterday (https://github.com/appcelerator/amp/pull/624/files). Did you rebase?  ;)